README.md

```diff
@@ -200,19 +200,53 @@ webboot very much: It lists a kernel, an initrd, and extra arguments to append.
 Before you continue, please make sure to meet the following conditions:
 
 - your system can boot from MBR (possibly through UEFI CSM)
-- you have a USB stick with MBR partitioning, first partition formatted as VFAT
-- you have a directory `/mnt/usb` to mount the partition to
-- you have `syslinux` installed (use your package manager)
-- you have built a suitable Linux kernel using a config from this repository
-- you have built an initramfs that includes webboot, gzip-compressed
+- You have a USB stick with MBR partitioning, first partition formatted as VFAT
```

Comment:
These last four steps are now part of your instructions further down, so it is a bit redundant to list them here.